templates/components/products/price.html

```diff
@@ -7,84 +7,80 @@
-                {{price.rrp_without_tax.formatted}}
-            </span>
-        </div>
-        <div class="price-section price-section--withoutTax non-sale-price--withoutTax{{#if price.with_tax}} price-section--minor{{/if}}" {{#unless price.non_sale_price_without_tax}}style="display: none;"{{/unless}}>
-            <span>
-                {{> components/products/price-label
-                    label_value=theme_settings.pdp-non-sale-price-label
-                    fallback=(lang "page_builder.pdp-non-sale-price-label")
-                }}
-            </span>
-            <span data-product-non-sale-price-without-tax class="price price--non-sale">
-                {{price.non_sale_price_without_tax.formatted}}
-            </span>
-        </div>
-        <div class="price-section price-section--withoutTax">
-            <span class="price-label" {{#if price.non_sale_price_without_tax}}style="display: none;"{{/if}}>
-                {{theme_settings.pdp-price-label}}
-            </span>
-            <span class="price-now-label" {{#unless price.non_sale_price_without_tax}}style="display: none;"{{/unless}}>
-                {{> components/products/price-label
-                    label_value=theme_settings.pdp-sale-price-label
-                    fallback=(lang "page_builder.pdp-sale-price-label")
-                }}
-            </span>
-            <span data-product-price-without-tax class="price price--withoutTax{{#if price.with_tax}} price-section--minor{{/if}}">{{price.without_tax.formatted}}</span>
-            {{#if price.with_tax}}
-                <abbr title="{{lang 'products.excluding_tax'}}">{{lang 'products.price_without_tax' tax_label=price.tax_label}}</abbr>
-            {{/if}}
-        </div>
-    {{/if}}
+    <div class="price-section price-section--withTax rrp-price--withTax" {{#unless price.rrp_with_tax}}style="display: none;"{{/unless}}>
+        <span>
+            {{> components/products/price-label
+                label_value=theme_settings.pdp-retail-price-label
+                fallback=(lang "page_builder.pdp-retail-price-label")
+            }}
+        </span>
+        <span data-product-rrp-with-tax class="price price--rrp">
+            {{price.rrp_with_tax.formatted}}
+        </span>
+    </div>
+    <div class="price-section price-section--withTax non-sale-price--withTax" {{#unless price.non_sale_price_with_tax}}style="display: none;"{{/unless}}>
+        <span>
+            {{> components/products/price-label
+                label_value=theme_settings.pdp-non-sale-price-label
+                fallback=(lang "page_builder.pdp-non-sale-price-label")
+            }}
+        </span>
+        <span data-product-non-sale-price-with-tax class="price price--non-sale">
+            {{price.non_sale_price_with_tax.formatted}}
+        </span>
+    </div>
+    <div class="price-section price-section--withTax price--withTax" {{#unless price.with_tax}}style="display: none;{{/unless}}>
```

Comment:
@bc-yevhenii-buliuk There is a missing quote at the end here.

Reply:
good catch, thanks